library/core/src/option.rs

```diff
@@ -1712,7 +1716,9 @@ impl<T> Option<T> {
     /// ```
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
-    #[rustc_const_unstable(feature = "const_option", issue = "67441")]
+    #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
+    #[rustc_allow_const_fn_unstable(const_replace)] // todo: drop if we don't get stable const_replace
```

Comment:
This is still in FCP at https://github.com/rust-lang/rust/pull/130954, so we have ti wait until tomorrow before we can land this PR as-is.

Reply:
That just merged so I was able to update & drop those gates.